src/main/java/bisq/core/dao/node/parser/TxInputParser.java

```diff
@@ -62,15 +62,13 @@ void process(TxInput txInput, int blockHeight, String txId, int inputIndex, Pars
                         case ISSUANCE_CANDIDATE_OUTPUT:
                             break;
                         case BLIND_VOTE_LOCK_STAKE_OUTPUT:
-                            if (parsingModel.getInputFromBlindVoteStakeOutput() == null) {
-                                parsingModel.setInputFromBlindVoteStakeOutput(txInput);
-                                // At the end of the parsing the OpReturnVoteRevealParser will verify that
-                                // this property is true, otherwise the tx will be considered invalid.
-                                parsingModel.setValidInputFromBlindVoteStakeOutput(true);
+                            if (parsingModel.getBlindVoteOutputState() == ParsingModel.BlindVoteOutputState.UNKNOWN) {
+                                // We found the input that's used as output for a blind vote. Only one such output is valid.
```

Comment:
Maybe better: We found the input that's used as output of a blind vote. Only one such output is valid.